Fachportal Padagogik.js

```diff
@@ -0,0 +1,313 @@
+{
+	"translatorID": "4b0b42df-76b7-4a61-91aa-b15bc553b77d",
+	"label": "Fachportal Pädagogik",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://www\\.fachportal-paedagogik\\.de/fis_bildung/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-01-05 22:54:10"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2016 Philipp Zumstein
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.indexOf('/suche/fis_set.html')>-1 || url.indexOf('/suche/fis_set_e.html')>-1) {
+		var coins = doc.getElementsByClassName("Z3988");
+		if (coins.length > 0) {
+			var info = coins[0].title;
+			if (info.indexOf("rft.genre=bookitem") > -1) {
+				return "bookSection";
+			}
+			if (info.indexOf("rft.genre=book") > -1) {
+				return "book";
+			}
+		}
+		return "journalArticle";
+	} else if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//div[contains(@class, "ergebnisliste")]/a');
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = new Array();
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+	var m = url.match(/FId=([\w\d]+)(&|$)/);
+	if (m) {
```

Comment:
wouldn't it make more sense to include the test for the FId in detectWeb? If we have any doubt that this isn't going to be there, we shouldn't be landing here.